main.tf

```diff
@@ -601,6 +639,63 @@ module "container_definition_bitbucket" {
   )
 }
 
+module "container_definition_azuredevops" {
+  source  = "cloudposse/ecs-container-definition/aws"
```

Comment:
because the inputs of these are copied and pasted, its possible for the logic to drift. i would copy the logic of each non pass through input and set as a local and then reuse across each container